www/docs/providers/keycloak.md

```diff
@@ -0,0 +1,40 @@
+---
+id: keycloak 
+title: Keycloak
+---
+
+## Documentation
+
+[Keycloak OIDC-Client configuration](https://www.keycloak.org/docs/6.0/server_admin/#oidc-clients)
+
+Make sure to set the URLs in the keycloak client configuration properly:
+
+* Base URL: http://localhost:3000/api/auth/callback/keycloak
+* Valid redirect URIs: http://localhost:3000/*
+
+## Example
+
+```dotenv
+# env.local
+KEYCLOAK_BASE_URL=http://localhost:8080/auth/realms/master/protocol/openid-connect
+KEYCLOAK_CLIENT_SECRET=yourclientsecret
+```
+
+```js
+// pages/api/auth/[...nextauth].js
+import Providers from 'next-auth/providers'
+
+...
+providers: [
+    Providers.Keycloak({
+        clientId: "nextjs_local",
```

Comment:
Is this some kind of convention, or would it be great to change this to `process.env.KEYCLOAK_CLIENT_ID` as other providers does that?
```suggestion
        clientId: process.env.KEYCLOAK_CLIENT_IT,
```

Reply:
no - thanks for the catch! just my local testing `clientId`
typo in the suggestion (`_IT` should be `_ID`) committed manually :)